src/resources/security.md

```diff
@@ -3,17 +3,31 @@ Security is an important topic to discuss. If security measures are disregarded,
 
 This article will share tips about how you can keep your bot and account safe.
 
-### Sharing Tokens
+### Sharing Tokens and Passwords
 **Do not share token(s) with anyone**. This includes both bot and regular user account tokens. Sharing your bot token with someone (or posting it publicly) grants them full-unlimited access to your editing bot. Sharing your user account token with someone (or posting it publicly) allows them to have full access to your account (even if they don't have your password or email). Once someone has your account or bot's token, there is a high chance of it being used for malicious purposes. For example, stealing personal info, spreading scams, modifying your bot to nuke/raid servers.
 
-The only thing you can do is regenerate your bot's token, but likely, the damage has already been done when you regenerate the token. You can't regenerate user account tokens, in such case you need to; you must contact Discord support for an optimal solution. So please, do not risk sharing your token(s) with anyone.
-The same applies to your account password, in such case you expose your account password; you should change it in user settings as soon as possible.
+In the event that your **bot's token** is shared, the only thing you can do is regenerate it, but by then, the damage has most likely been done. In the event you share your **account token** but still have access to your account, you can regenerate your token by changing your password. If you don't, you will need to contact Discord support for an optimal solution.
```

Comment:
```suggestion
In an event, if your **bot's token** is shared, the only thing you can do to secure it, is to regenerate a new bot token but by then, the damage has most likely been done. In the case of an **user account token**, if you still have access to your account, regenerate your token by changing your password. If you cannot or don't have access to it anymore, you will need to contact [Discord support](https://support.discord.com/hc/en-us/requests/new) for an optimal solution.
```

Reply:
This like is quite long, it should be split.
```suggestion
In the event, that your **bot's token** is shared, the only thing you can do to secure it is to regenerate the bot's token. But by then, the damage has most likely been done.
In the case of a **user account token**, if you still have access to your account, regenerate your token by changing your password. If you cannot or don't have access to it anymore, you will need to contact [Discord support](https://support.discord.com/hc/en-us/requests/new) for an optimal solution.
```